src/codec/id.rs

```diff
@@ -907,7 +923,7 @@ impl From<AVCodecID> for Id {
 			AV_CODEC_ID_HDMV_TEXT_SUBTITLE => Id::HDMV_TEXT_SUBTITLE,
 
 			/* other specific kind of codecs (generally used for attachments) */
-			AV_CODEC_ID_TTF       => Id::TTF,
+			AVCodecID_AV_CODEC_ID_TTF       => Id::TTF,
```

Comment:
Ah, because it makes it an actual enum, that's why they were just constants in the manual bindings.

Is there a way to ignore an enum field with bindgen? `AV_CODEC_ID_FIRST_UNKNOWN` and the other things should just be constants referring back to the enum.